tests/framework/extensions/corral/aws/config.go

```diff
@@ -0,0 +1,42 @@
+package aws
+
+import (
+	"reflect"
+
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+)
+
+// AWSCorralConfigConfigurationFileKey is the json/yaml config key for the an AWS corral
+const AWSCorralConfigConfigurationFileKey = "awsCorral"
+
+//AWSCorralConfig is aws struct for what is need for an AWS corral
+type AWSCorralConfig struct {
+	AWSRoute53Zone               string `json:"aws_route53_zone" yaml:"aws_route53_zone"`
+	AWSSecurityGroup             string `json:"aws_security_group" yaml:"aws_security_group"`
+	AWSAccessKkey                string `json:"aws_access_key" yaml:"aws_access_key"`
```

Comment:
There's a possible typo for the AWS access key field name

Reply:
fixed